vendors/vendor/boards/board/aws_demos/config_files/ota_demo_config.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
Product name in header?

```suggestion
* AWS IoT Device SDK for Embedded C V202012.00
 * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Reply:
Done. Updated other files to be consistent as well.